tdrs-frontend/src/components/IdleTimer/IdleTimer.jsx

```diff
@@ -0,0 +1,123 @@
+import React, { useRef, useState, useEffect } from 'react'
+import { useIdleTimer } from 'react-idle-timer'
+import { useDispatch } from 'react-redux'
+import Button from '../Button'
+import { fetchAuth } from '../../actions/auth'
+
+/**
+ * IdleTimer uses the `react-idle-timer` package to watch for user inactivity.
```

Comment:
Inline documentation. 👍 

It might be nice to name these env variables and/or where a dev should look to see where they are set.

Reply:
I just pushed an update to the documentation in `IdleTimer` for where the environment variables are stored and what they're used for.